bin/generate-pdf/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Wait... why _don't_ we have a global `node_modules` ignore in our root `.gitignore` file?